devicon.json

```diff
@@ -1562,6 +1562,29 @@
         "color": "#A80030",
         "aliases": []
     },
+    {
```

Comment:
The new icon needs to be in the correct alphabetical order compared to the other icons